utils/diffutils.xml

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" ?>
+<?xml-stylesheet type='text/xsl' href='interface.xsl'?>
+<interface uri="http://repo.roscidus.com/utils/diffutils" xmlns="http://zero-install.sourceforge.net/2004/injector/interface" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://zero-install.sourceforge.net/2004/injector/interface http://0install.de/schema/injector/interface/interface.xsd http://0install.de/schema/desktop-integration/capabilities http://0install.de/schema/desktop-integration/capabilities/capabilities.xsd">
+  <name>DiffUtils</name>
+  <summary xml:lang="en">show differences between files</summary>
+  <description xml:lang="en">You can use the diff command to show differences between two files, or each corresponding file in two directories. diff outputs differences between files line by line in any of several formats, selectable by command line options. This set of differences is often called a `diff' or `patch'. For files that are identical, diff normally produces no output; for binary (non-text) files, diff normally reports only that they are different. 
+
+You can use the cmp command to show the offsets and line numbers where two files differ. cmp can also show all the characters that differ between the two files, side by side. 
+
+You can use the diff3 command to show differences among three files. When two people have made independent changes to a common original, diff3 can report the differences between the original and the two changed versions, and can produce a merged file that contains both persons' changes together with warnings about conflicts. 
+
+You can use the sdiff command to merge two files interactively. 
+</description>
+  <icon href="https://raw.githubusercontent.com/0install/0install.de-feeds/master/Gow.ico" type="image/vnd.microsoft.icon"/>
+  <icon href="https://raw.githubusercontent.com/0install/0install.de-feeds/master/Gow.png" type="image/png"/>
+  <category>System</category>
+  <homepage>http://gnuwin32.sourceforge.net/packages/diffutils.htm</homepage>
+  <needs-terminal/>
+  <implementation arch="Windows-*" id="sha1new=b7ac58b6aac3d2e3e032305e667c5854818430be" langs="ca cs da de el es fi fr gl he hu id ja ms nl pl pt_BR ro ru sr sv tr zh_TW" license="GPL v2 (GNU General Public License)" released="2004-05-24" version="2.8.7-19-3">
```

Comment:
Best to use `arch="Windows-i486"` for EXEs that depend on 32-bit DLLs.

Reply:
Done